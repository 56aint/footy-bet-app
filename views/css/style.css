*{
  margin: 0;
  padding: 0;
}

.logo{
  color: blue;
  font-size: 25px;
  font-weight: bold;
  padding-left: 20px;
  margin-left: auto;
  margin-right: auto;
  width: 8em
}

ul{
  float: left;
  display: flex;
  justify-content: center;
  align-items: center;
}

ul li{
  list-style: none;
  margin: 62px;
  margin-top: 27px;
  font-size: 20px;

}

.container{
  position: absolute;
  padding: 16px;
  top: 30%;
  left: 50%;
  width: 50%;
  background-color:#FAFAFA;
  text-align: center;
  border-radius: 4px;
  text-transform: capitalize;
  text-align: center; 
  transform: translate(-50%,-50%);
}

h1 {
  background-color: rgb(45, 47, 187);
  color: #eef5ee;
  font-size: 2em;
  font-weight: lighter;
}

.title-box{
  margin: 25px 0;
  display: flex;
  flex-direction: row;
  flex-wrap: nowrap;
  justify-content: space-around;
  align-items: center;
  width: 100%;
  font-size: 1.5em;
}