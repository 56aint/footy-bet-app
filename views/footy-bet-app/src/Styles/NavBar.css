.navbar {
  background-color: rgb(49, 16, 197);
  overflow: hidden;
 /*  margin-left: 160px; */
  padding: 0 10px;
}
ul{
  float: left;
  display: flex;
  justify-content: center;
  align-items: center;
}

ul li{
  list-style: none;
  margin: 62px;
  margin-top: 27px;
  font-size: 20px;

}

.navbar a {
  color: white;
  font-weight: 700;
  letter-spacing: 2px;
  text-decoration: none;
  background: rgb(41, 38, 70);
  padding: 5px 0px;
  display: inline-block;
  width: auto;
  text-align: center;
  border-radius: 5px;
}

.navbar a:hover {
  color: #17bbd8;
}



.loading {
  margin: auto;
  width: auto;
  color: rgb(69, 9, 129);
  position: relative;
  text-align: center;
}
