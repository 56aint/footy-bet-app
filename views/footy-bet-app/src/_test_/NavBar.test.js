import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import NavBar from '../Component/NavBar.';
import '../Styles/NavBar.css';

describe('NavBar', () => {
  test('renders correctly', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <NavBar />
      </BrowserRouter>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
