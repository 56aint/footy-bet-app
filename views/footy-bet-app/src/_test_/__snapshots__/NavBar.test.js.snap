// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <ul
      class="navbar-links"
      data-testid="navbar-links-id"
    >
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          class="navbar-item-home"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          class="navbar-item-myTeam"
          href="/my-favourite-team"
        >
          My Team
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          href="/"
        >
          Live Football
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          href="/"
        >
          Market
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          href="/"
        >
          Outcomes
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          href="/"
        >
          Market Updates
        </a>
      </li>
      <li
        class="navbar-links-item"
        data-testid="navbar-links-item-id"
      >
        <a
          href="/"
        >
          Outcomes Updates
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
